messages/delete.md

```diff
@@ -1,14 +1,15 @@
 # summary
 
-Delete a scratch org or sandbox.
+Delete a scratch or sandbox org.
 
 # deprecation
 
 The force:org:delete command is deprecated. Use org:delete:scratch or org:delete:sandbox.
 
 # description
 
-mark a scratch or sandbox org for deletion
+Deleting a scratch or sandbox org is a two-part process. Salesforce CLI first deletes all local references to the org from your computer. It then it marks the org for deletion in either the Dev Hub org (for scratch orgs) or production org (for sandboxes.)
```

Comment:
technically, we mark it in the org first before removing the connection (otherwise we'd not be able to connect).  

Text here makes it sounds sequential.

```suggestion
Deleting a scratch or sandbox org is a two-part process. Salesforce CLI first deletes all local references to the org from your computer. It also marks the org for deletion in either the Dev Hub org (for scratch orgs) or production org (for sandboxes.)
```

Reply:
Better?